### ISSUE TITLE
Exporting to Hearthstone Failure - 1 Character Searches

### ISSUE THREAD

**Issue by ArmageddonX430** [HearthSim/Hearthstone-Deck-Tracker#2863]
I haven't been able to export a previously created deck into Hearthstone automatically for a few weeks now.  Everything works normally except the search field only receives the first character of the string literal.  This results in the deck tracker trying to add Arcane Shot, Bestial Wrath, and Brave Archer about seven times each, and nine other randomly lettered early hunter cards.

**Comment by azeier** [comment]
Try playing around with the settings under `options (advanced) > tracker > exporting`. Specifically `speed` and `paste names from clipboard`.

**Comment by ArmageddonX430** [comment]
Yup, I messed with those settings with great fervor and determination!  Sorry I forgot to mention it.  The only difference is when I was pasting names from clipboard, literally nothing showed up in the search field.  The speed changes changed the speed as would be expected.

**Comment by azeier** [comment]
- Any difference when running Hearthstone in windowed/fullscreen?
- Could be an issue with software like RadeonPro - that is known to cause issues when exporting in some cases.

Other than those I don't really have ideas what might cause this.

**Comment by ArmageddonX430** [comment]
Well I just changed Hearthstone to windowed mode... and it worked!  Then I set it back to fullscreen... and it still works!  I don't actually think that was the issue though.  As much as I hate saying, "did you try turning it off and on again?" I recently had a power-out and had to boot up my computer again.  I leave my computer running 24/7.  So, the reboot may have solved it.

Also, I do use Radeon cards (MSI 7970x2), but it does not look like I have "RadeonPro" installed.  Just "AMD Radeon Settings".
